### PR TITLE
Regenerate audio file if API information for artwork has changed.

### PR DESCRIPTION
**Pull request by indhu-gunda**

### CODE REVIEW

File: backend/src/main/java/com/google/capstone/TextToSpeech.java [hunk 1]
```diff
@@ -59,11 +60,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
+  private boolean inStorage(String objectIdString, String textString) {
+    Blob blob = storage.get(BlobId.of(BUCKET_NAME, objectIdString));
+    if (blob != null) {
+      String transcript = blob.getMetadata().get("audioTranscript");
```
Comment (BearBurg) [inline backend/src/main/java/com/google/capstone/TextToSpeech.java:66]: More often people use a hash function to produce a [fingerprint](https://en.wikipedia.org/wiki/Fingerprint_(computing)) to save storage space.

Comment (indhu-gunda) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:66]: is there a hash function or an algorithm you recommend for generating this?

Comment (indhu-gunda) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:66]: Should I use String.hashcode()?

Comment (BearBurg) [reply backend/src/main/java/com/google/capstone/TextToSpeech.java:71]: Discussed offline.

**Pull request merged**